x-pack/plugins/maps/server/maps_telemetry/collectors/register.ts

```diff
@@ -7,7 +7,7 @@
 import { UsageCollectionSetup } from 'src/plugins/usage_collection/server';
 import { getMapsTelemetry } from '../maps_telemetry';
 // @ts-ignore
-import { TELEMETRY_TYPE } from '../../../common/constants';
```

Comment:
I;d change the `TELEMETRY_TYPE` constant for clarity https://github.com/elastic/kibana/blob/55a73616ab706b4588e4c681a1369c2f85c1f907/x-pack/plugins/maps/common/constants.ts#L28

e.g.:

```export const TELEMETRY_TYPE = APP_ID;```

If keeping it like this, then definitely remove the unused TELEMETRY_TYPE variable.

Reply:
Yeah I see it, that was my mistake :disappointed:

Reply:
Not blaming! It actually ended up being useful because it's easier to rename `maps-telemetry.attributes` to `maps` than renaming `maps-telemetry.attributes` to `maps-telemetry` because of the way the [`rename` ingest pipeline processor](https://www.elastic.co/guide/en/elasticsearch/reference/current/rename-processor.html) works 😄

Reply:
haha thanks @afharo, great to get the confirmation and go-ahead on this :)